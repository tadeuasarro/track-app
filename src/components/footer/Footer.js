import { Link } from 'react-router-dom';
import './footer.css';

const Footer = () => (
  <div className="footer-container">
    <Link className="footer-link" to="/measures">
      <i className="fas fa-balance-scale-right" />
    </Link>
    <Link className="footer-link" to="/">
      <i className="fas fa-bars" />
    </Link>
    <Link className="footer-link" to="/progress">
      <i className="fas fa-tasks" />
    </Link>
    <Link className="footer-link" to="/more">
      <i className="fas fa-info" />
    </Link>
  </div>
);

export default Footer;
