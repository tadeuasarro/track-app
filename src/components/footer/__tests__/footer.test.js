import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('Display', () => {
  it('renders correctly', () => {
    const comp = renderer.create(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    ).toJSON();
    expect(comp).toMatchSnapshot();
  });
});
