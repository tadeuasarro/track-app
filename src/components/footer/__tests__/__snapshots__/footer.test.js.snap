// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `
<div
  className="footer-container"
>
  <a
    className="footer-link"
    href="/measures"
    onClick={[Function]}
  >
    <i
      className="fas fa-balance-scale-right"
    />
  </a>
  <a
    className="footer-link"
    href="/"
    onClick={[Function]}
  >
    <i
      className="fas fa-bars"
    />
  </a>
  <a
    className="footer-link"
    href="/progress"
    onClick={[Function]}
  >
    <i
      className="fas fa-tasks"
    />
  </a>
  <a
    className="footer-link"
    href="/more"
    onClick={[Function]}
  >
    <i
      className="fas fa-info"
    />
  </a>
</div>
`;
